packages/babel-jest/src/__tests__/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
These are just general `tsconfig.json`. Few test suites need tiny fixing. I though to start with adding base `tsconfig.json` with `"types": ["jest"]` for all of them. That’s a lot of files added. Keeping it the same and simple should speed up the review. Later I will come back with with fix PRs case-by-case.